plugins/robothor_plugin/robothor_viz.py

```diff
@@ -145,25 +150,25 @@ def make_top_down_views(self) -> Dict[str, np.ndarray]:
             if not os.path.exists(fname):
                 if self.controller is None:
                     self.controller = Controller()
-                    self.controller.step(
-                        {"action": "ChangeQuality", "quality": "Very High"}
-                    )
-                    self.controller.step(
-                        {
-                            "action": "ChangeResolution",
-                            "x": self.viz_rows_cols[1],
-                            "y": self.viz_rows_cols[0],
-                        }
-                    )
+                self.controller.step(
+                    {"action": "ChangeQuality", "quality": "Very High"}
+                )
+                self.controller.step(
+                    {
+                        "action": "ChangeResolution",
+                        "x": self.viz_rows_cols[1],
+                        "y": self.viz_rows_cols[0],
+                    }
+                )
 
                 self.dump_top_down_view(roomname, fname)
             top_downs[roomname] = cv2.imread(fname)
 
         return top_downs
 
     def crop_viz_image(self, viz_image: np.ndarray) -> np.ndarray:
-        y_min = 0
-        y_max = int(self.viz_rows_cols[0] * 0.5)
+        y_min = int(self.viz_rows_cols[0] * 0.27)
+        y_max = int(self.viz_rows_cols[0] * 0.77)
```

Comment:
Yes. They are not very good anyway, because the resulting image still shows the room too close to the bottom of the image. I'll try one or two more values and add a comment.